### DISCUSSION TITLE
Large postgres database

### DISCUSSION THREAD

**Discussion by levi770** [vercel/community#2376] [Help] [answered]
### Summary

Hello!
i have a large postgres database 13 GB, is it possible to store it with vercel postgres database?

**Comment by amyegan** [comment, marked as answer]
Vercel Postgres is better suited to smaller databases at the moment, with a Hobby account limit of 256 MB stored data and a Pro plan limit of 512 MB with the option to pay per additional GB used.

For larger databases, it may be better to look at one of our Postgres integration partners, such as [Supabase](https://vercel.com/integrations/supabase), [Cockroach DB](https://vercel.com/integrations/cockroachdb), or [Thin Backend](https://vercel.com/integrations/thin). Or stick with your current solution if you already have something that works well for you 🙂 

You can find more information about Vercel Postgres limits at https://vercel.com/docs/storage/vercel-postgres/usage-and-pricing
